src/main/java/net/tylers1066/beaming/commands/AbandonShipCommand.java

```diff
@@ -0,0 +1,40 @@
+package net.tylers1066.beaming.commands;
+
+import net.countercraft.movecraft.craft.Craft;
+import net.countercraft.movecraft.craft.PlayerCraft;
+import net.countercraft.movecraft.craft.type.CraftType;
+import net.tylers1066.beaming.Beaming;
+import net.tylers1066.beaming.localisation.I18nSupport;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+public class AbandonShipCommand implements CommandExecutor {
+
+    @Override
+    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command cmd, @NotNull String s, @NotNull String[] args) {
+        if (!cmd.getName().equalsIgnoreCase("abandonship")) {
+            return false;
+        }
+
+        if (!(sender instanceof Player)) {
+            sender.sendMessage(Beaming.PREFIX + I18nSupport.getInternationalisedString("Players Only"));
+            return true;
+        }
+
+        if (Beaming.getInstance().getCrewCrafts().containsKey((Player)sender)) {
+            Craft craft = Beaming.getInstance().getCrewCrafts().get(sender);
+
+            if (craft instanceof PlayerCraft) {
```

Comment:
Does this need to be a player craft, or could somebody crew a piloted craft (like a squadron craft)?